backend/apps/owasp/models/project_health_requirements.py

```diff
@@ -57,6 +58,12 @@ class Meta:
     unassigned_issues_count = models.PositiveIntegerField(
         verbose_name="Unassigned issues", default=0
     )
+    compliance_penalty_weight = models.FloatField(
```

Comment:
It's out of the feature scope. We have the scoring system already in place and there is no need to introduce additional fields.